wikimedia/wikipedia-ios
# Image recommendations - Explore feed card (Part 1)

**Phabricator:**
https://phabricator.wikimedia.org/T356811

### Notes
This PR lays the groundwork for the suggested edits Explore feed card. The sorting logic and UI will be finalized in followup PRs.

### Test Steps
1. Fresh install Wikipedia app on Staging.
2. Confirm you see a dummy suggested edits card after Featured Article.
3. Confirm tapping card sends you to dummy image recommendations view controller.
4. Go to Explore preferences and disable suggested edits. Confirm card is now hidden.
5. Re-enable in Explore preferences. From card tap the 3-dot icon and hide card. Confirm card is hidden. Go to Explore preferences and confirm toggle is off.
6. Fresh install main Wikipedia app scheme. Confirm card does not display.
7. Confirm notifications center is still reachable as before.

### Screenshots
![screenshot3](https://github.com/wikimedia/wikipedia-ios/assets/3620196/a522f1d3-7daf-4f41-a770-9a90370b3ba6)
